pages/cards/dailycard/detail/notes/index.js

```diff
@@ -0,0 +1,5 @@
+import NoteFormular from "../../../../../components/notes/notes";
+
+export default function Notes({}) {
+  return <NoteFormular></NoteFormular>;
+}
```

Comment:
This seems redundant - you could use `NoteFormular` directly instead of wrapping it into `Notes`